src/main/java/com/damagecounter/DamageCounterPlugin.java

```diff
@@ -376,8 +418,9 @@ private void reset()
 				double damageTotal = total.getDamage();
 				double damagePercent = damageDone / damageTotal;
 				DecimalFormat df = new DecimalFormat("##%");
-				if (damagePercent < 1)
-				{
+				if(killStatus.equals(KillStatus.UNSUCCESSFUL)){
+					client.addChatMessage(ChatMessageType.GAMEMESSAGE, "", "You dealt " + QuantityFormatter.formatNumber(damageMember.getDamage()) + " damage in " + killTime, null);
```

Comment:
Ran into an annoying case where since we didn't kill the enemy, we don't have it's name. I'm currently deciding if we should just grab the name earlier so we don't need a special case here or not.